package.json

```diff
@@ -29,12 +29,14 @@
     "coveralls": "2.11.15",
     "css-loader": "1.0.0",
     "d3": "^5.7.0",
+    "istanbul-instrumenter-loader": "^3.0.1",
     "jasmine": "2.5.2",
     "jasmine-jquery": "2.1.1",
     "jquery": "3.2.1",
     "karma": "2.0.4",
     "karma-chrome-launcher": "2.2.0",
     "karma-coverage": "1.1.2",
+    "karma-coverage-istanbul-reporter": "^2.0.4",
```

Comment:
Are old package still in use? Otherwise, let remove useless packages from dependencies

Reply:
Done!